HGGM/appsettings.json

```
@@ -2,5 +2,11 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "LiteDb": "Filename=Database.db"
-  } 
+  },
+  "EmailSettings": {
+    "Email": "Test",
```

```suggestion
    "Email": "noreply@example.com",
```
and so on for others